packages/flutter/lib/src/material/app.dart

```diff
@@ -263,11 +263,6 @@ class MaterialApp extends StatefulWidget {
 
 class _MaterialScrollBehavior extends ScrollBehavior {
```

Comment:
not doing that is intentional, to verify that we can in fact do this in two layers